bin/ember

```diff
@@ -27,5 +27,7 @@ resolve('ember-cli', {
     cliArgs: process.argv.slice(2),
     inputStream: process.stdin,
     outputStream: process.stdout
-  }).then(exit);
+  }).then(function(result) {
+    exit(result && result.exitCode || result);
```

Comment:
Actually, I think we need to handle falsey exit codes `0` is an exit code and is falsey.

Something like:

```javascript
var exitCode = typeof result === 'object' ? result.exitCode : result;

exit(exitCode);
```